library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -418,6 +418,14 @@ return_status libspdm_build_response(IN void *context, IN uint32_t *session_id,
             status = RETURN_NOT_FOUND;
         }
     }
+    /*if return the status: Responder drop the response*/
+    /* just ignore this message */
+    /* return UNSUPPORTED and clear response_size to continue the dispatch without send response.*/
```

Comment:
Please add above comment to "libspdm_get_response_func" API definition in "spdm_responder_lib.h"

Reply:
Yes. I have added the comment.